src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/Version.java

```diff
@@ -6,6 +6,6 @@
 
 class Version {
     public static String getDetailedVersion() {
-        return "99.99.99-dev";
+        return "5.0.0-dev";
```

Comment:
Isn't this file automatically updated anyhow? I'm wondering because PR validation (where the version is "5.0.0-ci") and builds of `release/5.0-preview*` have been succeeding.

Put another way, does this file need to change?

Reply:
If we don't update the file then local builds will constantly show this file as changed which is annoying. We _could_ ignore the file, but this seems fine.